apps/music/test/marionette/playlist_test.js

```diff
@@ -245,5 +245,37 @@ marionette('Music player playlist', function() {
       assert.equal(PlaylistHelper.songTitle(songs[1]), 'Crash');
     });
 
+    test('Recently added playlist sort order', function() {
+      // start the app so the music files are added to the database.
+      music.launch();
+      music.waitForFirstTile();
+
```

Comment:
Once you figure out how to wait for scanning to be complete, you should probably do that here to be sure that the test is robust.